src/workflow/ArcanistDiffWorkflow.php

```diff
@@ -443,6 +443,9 @@ public function getArguments() {
           'advice'    => pht('%s suppresses lint.', '--head'),
         ),
       ),
+      'trace-git-push' => array(
```

Comment:
Please add some comment - it is Uber code here. This will let us avoid some problems with merging upstream code to this repo.
```
// START Uber code
...
// END Uber code
```